src/app/components/search/search.component.scss

```diff
@@ -1,4 +1,12 @@
+.govuk-error-summary__body a {
+  color: #d4351c;
```

Comment:
Maybe put this color in var at the top level?

Reply:
Yes I'm not sure where this came from but I can move it